src/pages/Cart.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react'
+import CartItem, { CartItemType } from '../components/CartItem/CartItem'
+import { PaymentInfo } from '../components/PaymentInfo'
+import { useApiClient } from '../context/ApiClientContext'
+
+function Cart() {
+  const { cartHttpClient } = useApiClient()
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState(null)
+  const [carts, setCarts] = useState<CartItemType[] | []>([])
```

Comment:
빈 배열은 어떠한 요소도 포함하지 않은 배열이고, `CartItemType[]`은 배열의 모든 요소가 `CartItemType`이어야 한다는 제약인데요.

그렇기 때문에 아래와 같이 타입을 지정해주셔도 됩니다!
```suggestion
  const [carts, setCarts] = useState<CartItemType[]>([])
```